packages/app/src/app/pages/Dashboard/Content/routes/Settings/components/Navigation.tsx

```diff
@@ -57,10 +58,12 @@ export const SettingNavigation = (props: SettingsNavigationProps) => (
         url={dashboardUrls.settings(props.teamId)}
         label="Workspace"
       />
-      <NavigationLink
-        url={dashboardUrls.registrySettings(props.teamId)}
-        label="Npm Registry"
-      />
+      {!props.isPersonal && (
+        <NavigationLink
+          url={dashboardUrls.registrySettings(props.teamId)}
+          label="Npm Registry"
```

Comment:
super silly pick, but 
```suggestion
          label="npm Registry"
```

Reply:
Oh yes I've been thinking about this! So npm needs to be written without capitals, but when it's at the start of a sentence, shouldn't then the first letter be capitalized like any other word? Not sure, I at least do like the capital more from a design perspective (consistency), but that shouldn't be the reason to do it.

Reply:
That's technically right, yes, but the mixed case does look wrong here somehow. Try all upper case e.g. NPM Registry - that will visually look better and folks will still know what you mean.